packages/experimental/Shimmer/src/Shimmer.win32.tsx

```diff
@@ -0,0 +1,100 @@
+/** @jsx withSlots */
+import { compose, mergeProps, UseSlots } from '@fluentui-react-native/framework';
+import { View } from 'react-native';
+import { ClipPath, Defs, LinearGradient, Path, Rect, Stop, Svg } from 'react-native-svg';
+import { styleSettings } from './Shimmer.styling.win32';
+import { shimmerName, ShimmerProps, ShimmerTokens } from './Shimmer.types';
+import { ClippingMaskProps, ShimmerType } from './Shimmer.types.win32';
+import { RCTNativeAnimatedShimmer } from './consts.win32';
+import { convertRectToSvgPath, convertCircleToSvgPath } from './SvgShapeToPath';
+import { withSlots } from '@fluentui-react-native/framework';
+
+/** Absolute positioning is used to overlay the clipping mask on top of the shimmer wave. */
+const clippingMask: React.FunctionComponent<ClippingMaskProps> = (props: ClippingMaskProps) => {
+  return (
+    <Svg style={{ position: 'absolute' }} width="100%" height="100%">
+      <Defs>
+        <ClipPath id="shimmerCuts">
+          <Path d={props.clipPath} clipRule="evenodd" />
+        </ClipPath>
+      </Defs>
+
+      <Rect width="100%" height="100%" fill={props.backgroundColor as any} clipPath="url(#shimmerCuts)" />
+    </Svg>
+  );
+};
+
+const wave: React.FunctionComponent<ShimmerTokens> = (props: ShimmerTokens) => {
+  const shimmerColor: any = props.shimmerColor;
+  const shimmerWaveColor: any = props.shimmerWaveColor;
+
+  return (
+    <Svg {...props} width="100%" height="100%">
+      <LinearGradient id="gradient">
+        <Stop stopColor={shimmerColor} />
+        <Stop offset="20%" stopColor={shimmerWaveColor} />
+        <Stop offset="40%" stopColor={shimmerColor} />
+      </LinearGradient>
+      <Rect width={props.shimmerWaveWidth} height="100%" fill="url(#gradient)" fillOpacity={props.gradientOpacity} />
+    </Svg>
+  );
+};
+
+const waveContainer: React.FunctionComponent<ShimmerTokens> = (props: ShimmerTokens) => {
+  return <RCTNativeAnimatedShimmer {...{ ...props, style: { backgroundColor: props.shimmerColor, overflow: 'hidden' } }} />;
+};
+
+export const Shimmer = compose<ShimmerType>({
+  displayName: shimmerName,
+  ...styleSettings,
+
```

Comment:
nit: I think we normally call this stylingSettings? not a big deal though, that name swaps out between the old and new framework anyway.